src/microbe_stage/systems/StrainSystem.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace Systems
+{
+    using Components;
+    using DefaultEcs;
+    using DefaultEcs.System;
+    using DefaultEcs.Threading;
+    using Godot;
+    using World = DefaultEcs.World;
+
+    [With(typeof(CompoundStorage))]
+    [With(typeof(OrganelleContainer))]
+    [With(typeof(MicrobeControl))]
+    [With(typeof(StrainAffected))]
+    [RunsAfter(typeof(PhysicsBodyDisablingSystem))]
```

Comment:
Why does this run after the body disabling system?

Reply:
The logic behind this was that since this is supposed to run before `MicrobeMovementSystem`, and that system ran after `PhysicsBodyDisablingSystem`, I moved the `RunsAfterAttribute` to `StrainSystem`, but looking at it now it might not be correct.

Reply:
Then set this directly to run after the movement system. The threaded run generator can handle recursive dependencies so A -> B -> C runs after attributes will ensure that A will only run after C.